channeldb/graph.go

```diff
@@ -3616,9 +3669,34 @@ func putChanEdgePolicy(edges, nodes *bbolt.Bucket, edge *ChannelEdgePolicy,
 		return err
 	}
 
+	updateEdgePolicyDisabledIndex(edges, edge.ChannelID,
+		edge.ChannelFlags&lnwire.ChanUpdateDirection > 0, edge.IsDisabled())
+
 	return edges.Put(edgeKey[:], b.Bytes()[:])
 }
 
+func updateEdgePolicyDisabledIndex(edges *bbolt.Bucket, chanID uint64,
+	initiator bool, disabled bool) error {
```

Comment:
is `initiator` the appropriate name? seems instead like this just the direction, which is determined by comparing the node pubkeys

Reply:
Agree. Fixed.